devsetup/tripleo/tripleo_install.sh

```diff
@@ -15,69 +15,158 @@
 # under the License.
 set -ex
 
-openstack undercloud install
+openstack undercloud install --no-validations
 source stackrc
 
-openstack overcloud network provision --output network_provision_out.yaml ./network_data.yaml
-openstack overcloud network vip provision --stack overcloud --output vips_provision_out.yaml ./vips_data.yaml
+# TODO(bogdando): cells-specific DNS domains and neworks in network v2 data
+if [ $EDPM_COMPUTE_CELLS -gt 1 ] ; then
+    cp overcloud_services_cell0.yaml overcloud_services.yaml
+    # For the time being, use the same network data (with network leafs) for all stacks we deploy
+    for cell in $(seq 0 $(( EDPM_COMPUTE_CELLS - 1))); do
+        [ $cell -eq 1 ] && continue
+        cp network_data${cell}.yaml network_data${cell}_cells_specific_unused.yaml
+        cp network_data1.yaml network_data${cell}.yaml
+    done
+    sed -i "s/overcloud/multistack/g" overcloud_services.yaml
+fi
+
+openstack overcloud network provision --output network_provision_out0.yaml ./network_data0.yaml
```

Comment:
why these filenames out0.yaml please restore?

Reply:
this made it simpler to iterate for multi-stack, where is stack 0 is overcloud.
I will retain the name via adding an alias to the old names for a single-stack multi-node compat